web/src/modules/topic/components/Score/Score.tsx

```diff
@@ -31,82 +31,83 @@ interface ScoreProps {
-          <StyledButton
-            onClick={() => setSelected(true)}
-            onMouseEnter={() => setHovering(true)}
-            buttonLength={buttonLength}
-            variant="contained"
-            color={scoreColor}
-            ref={mainButtonRef}
-          >
-            {/* i bet material icons would look way nicer than this text... but material only has number icons 1-6 :( */}
-            {score}
-          </StyledButton>
-
-          {/* jank: zoom needs to be manually applied to the popper & its children because it is not a child of the flow element */}
-          {/* cannot just apply scale to Popper because Popper sets transform in styles in order to position next to anchor, and scale affects that */}
-          <StyledPopper
-            id="simple-popper"
-            onMouseLeave={() => setHovering(false)}
-            modifiers={[
-              {
-                name: "offset",
-                options: {
-                  // position centered on top of the main button https://popper.js.org/docs/v2/modifiers/offset/
-                  offset: [0, -1 * ((pieDiameter * zoom) / 2 + (buttonLength * zoom) / 2)],
-                },
-              },
-            ]}
-            open={hovering || selected}
-            anchorEl={mainButtonRef.current}
-          >
-            <ScorePie
-              pieDiameter={pieDiameter * zoom}
-              arguableId={arguableId}
-              arguableType={arguableType}
-            />
+        <CenterCircleDiv>
+          <ScorePie
+            circleDiameter={circleDiameter}
+            arguableId={arguableId}
+            arguableType={arguableType}
+          />
+        </CenterCircleDiv>
 
-            {/* second button here because we want Pie to display in front of everything except the button... user isn't supposed to know there's two */}
-            {/* TODO: extract ScoreButton component for reuse - couldn't figure out how to get forwardref to work */}
-            <StyledButton
-              onClick={() => setSelected(true)}
-              onMouseEnter={() => setHovering(true)}
-              buttonLength={buttonLength}
-              variant="contained"
-              color={scoreColor}
-              style={{
-                position: "absolute",
-                left: "50%",
-                top: "50%",
-                transform: `translate(-50%, -50%) scale(${zoom})`,
-              }}
-            >
-              {/* i bet material icons would look way nicer than this text... but material only has number icons 1-6 :( */}
-              {score}
-            </StyledButton>
-          </StyledPopper>
-        </Box>
-      </ClickAwayListener>
-    </>
+        {/* second button here because we want Pie to display in front of everything except the button... user isn't supposed to know there's two */}
+        {/* TODO: extract ScoreButton component for reuse - couldn't figure out how to get forwardref to work */}
+        <StyledButton
```

Comment:
`scale` was used on the button style because the font also needs to be scaled - right now this button's font is no longer being scaled